mmv1/third_party/terraform/utils/provider_test.go.erb

```diff
@@ -658,11 +658,13 @@ func TestAccProviderIndirectUserProjectOverride(t *testing.T) {
 func testAccProviderBasePath_setBasePath(endpoint, name string) string {
 	return fmt.Sprintf(`
 provider "google" {
+  alias                   = "compute_custom_endpoint"
   compute_custom_endpoint = "%s"
 }
 
 resource "google_compute_address" "default" {
-	name = "address-test-%s"
+	provider = google.compute_custom_endpoint
+	name     = "address-test-%s"
```

Comment:
Spacing is a bit off here. We use spaces rather than tabs in configuration.

Reply:
Changed. Thanks.